### PR TITLE
Dev v1.0.6

### PR DESCRIPTION
**Pull request by FoundationKen**

### CODE REVIEW

**Review by coreylakey — APPROVED** [review]
I approve.

Small misspell on line 733, if worth it I can make the change before the pull request.

File: ports/stm32/boards/Passport/modules/new_wallet.py [hunk 1]
```diff
@@ -714,16 +715,24 @@ async def show(self):
...
+                    save_next_addr(self.acct_num, self.addr_type, addr_idx, is_change)
+                    address = format_btc_address(address, self.addr_type)
+
+                    result = await ux_show_story('''{}
+
+Found at index: {}\nType: {}'''.format(address, addr_idx, 'Change' if is_change == 1 else 'Receive'), title='Verified', left_btn='BACK',
+                                                 right_btn='CONTINUE', center=True, center_vertically=True)
+                    if result == 'x':
+                        if not self.goto_prev():
+                            # Nothing to return back to, so we must have skipped one or more steps...were' done
```
Comment (coreylakey) [inline ports/stm32/boards/Passport/modules/new_wallet.py:733]: Misspelled "were'".

Comment (FoundationKen) [reply ports/stm32/boards/Passport/modules/new_wallet.py:733]: Fixed

**Review by coreylakey — APPROVED** [review]
Approved :+1:

**Pull request merged**